### ISSUE TITLE
redis::aio::ConnectionLike` is not satisfied

### ISSUE THREAD

**Issue by losintikfos** [fakeshadow/tang_rs#6]
Any suggestions on how I can resolve this error:

```
the trait bound `redis::aio::MultiplexedConnection: redis::aio::ConnectionLike` is not satisfied
perhaps two different versions of crate `redis` are being used?
the following other types implement trait `redis::aio::ConnectionLike`:
  MultiplexedConnection
  redis::aio::Connection<C>
```

I have upgraded my redis to:

```
redis = { version = "0.21.5", features = ["async-std-comp", "aio"] }
redis_tang = { git = "https://github.com/fakeshadow/tang_rs.git", branch = "lock-free" }
```

**Comment by fakeshadow** [comment]
redis_tang depend on redis 0.19. I just released 0.3 version to bump the dependency.
You can use `redis_tang = "0.3"` to import it.

**Comment by losintikfos** [comment]
Thanks so much
